website/content/en/highlights/2022-02-08-0-20-0-upgrade-guide.md

```diff
@@ -115,6 +116,26 @@ if contains(.message, "true") ?? false {
 }
 ```
 
+#### Division by a literal nonzero number in `VRL` is no longer fallible {#vrl-infallible-division}
```

Comment:
Is this right? I think you copied the description from #10339

Reply:
Ah, this PR was based on that PR, I don't think it has picked up that this change was a part of that PR and not this PR. I'll merge master in and add a proper upgrade - it looks like we may have time to sneak this into 0.20 too.

Reply:
Thanks! And yeah, I think we do have time since we are still blocked on https://github.com/vectordotdev/vector/issues/11245